src/datatypes/gadgets/Gadget.cpp

```diff
@@ -86,6 +86,6 @@ namespace spy::gadget {
     }
 
     bool Gadget::operator==(const Gadget &rhs) const {
-        return std::tie(type, usagesLeft) == std::tie(rhs.type, rhs.usagesLeft);
+        return this->type == rhs.type;
```

Comment:
Warum die Gleichheit nur über den Typ?

Reply:
Habe in der LibClient eine map und da hätte ich gerne das der key nur nach type verglichen wird. Da jedes Gadget sowieso nur einmal vorkommt (bis auf Cocktails, aber die haben eh keine explitzite Vergelichsmethode) sollte das auch nicht weiter schlimm sein.